### PR TITLE
fix: issue #475

### PR DESCRIPTION
**Pull request by yoution**
@maxceem please review

### CODE REVIEW

File: app-constants.js [hunk 1]
```diff
@@ -5,9 +5,12 @@
 const UserRoles = {
   BookingManager: 'bookingmanager',
   Administrator: 'administrator',
-  ConnectManager: 'Connect Manager'
+  ConnectManager: 'Connect Manager',
+  TopcoderUser: 'Topcoder User'
 }
 
+const TopCoderUserPermissionRole = UserRoles.TopcoderUser
```
Comment (maxceem) [inline app-constants.js:12]: I guess there is no need to export this role separately, can use it as `UserRoles.TopcoderUser` inside the code

File: src/services/JobCandidateService.js [hunk 2]
```diff
@@ -358,11 +360,50 @@ searchJobCandidates.schema = Joi.object().keys({
...
+        jobCandidateUserHandle: handle,
+        jobName: job.title,
+        notificationType: {
+          jobCandidateResumeViewed: true
+        }
+      }
+    })
+
+    await updateJobCandidate(currentUser, jobCandidate.id, { viewedByCustomer: true })
+  }
```
Comment (maxceem) [inline src/services/JobCandidateService.js:391]: Let's wrap it in `try / catch` and if some error happened during email notification sending we just log error, but still error resume downloading.

**Review by maxceem — CHANGES_REQUESTED** [review]
Thank you @yoution. It works good, I only would like to make some updates for the code.

1. The way it is implemented now, the resume would be downloaded by the TaaS API and then sent to the client by the TaaS API. Can we avoid downloading resume TaaS API, and just redirect to the resume like this https://expressjs.com/en/api.html#res.redirect ?

2. Let's add a title to the email:

    image

Also, please check comments to the code below.

**Comment by yoution** [comment]
@maxceem 
`Let's add a title to the email:`, the html is using `table` layout, it is a little complex, is there any web tools to automatically create the layout?

**Comment by maxceem** [comment]
@yoution we already support title, just pass `description: "Client Viewed Resume"` inside the email data and it would be inserted inside email here https://github.com/topcoder-platform/taas-apis/blob/feature/notifications-api/data/notifications-email-template.html#L75.

**Review by maxceem — APPROVED** [review]
thank you @yoution works good as per local testing.

**Pull request merged**